xml/System.Text.RegularExpressions/RegexParseException.xml

```diff
@@ -59,10 +58,10 @@
         <Parameter Name="context" Type="System.Runtime.Serialization.StreamingContext" />
       </Parameters>
       <Docs>
-        <param name="info">To be added.</param>
-        <param name="context">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="info">The object that holds the serialized object data.</param>
+        <param name="context">The contextual information about the source or destination.</param>
+        <summary>Sets the <paramref name="info" /> object with the parameter name and additional exception information.</summary>
```

Comment:
@pgovind the method is `GetObjectData`. I think it wouldn't be accurate if the summary says it "sets" something.

Reply:
No I think this is correct. I picked it from the base class: https://docs.microsoft.com/en-us/dotnet/api/system.argumentexception.getobjectdata?view=netcore-3.1